Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -115,6 +115,14 @@ public final class ManifestLoader {
         let toml = try TOMLItem.parse(tomlString)
         let package = PackageDescription.Package.fromTOML(toml, baseURL: baseURL)
         let products = PackageDescription.Product.fromTOML(toml)
+        let errors = parseErrors(toml)
+
+        guard errors.isEmpty else {
+            // FIXME: Output with proper diagnostics infrastucture when available.
+            print("Found errors while parsing manifest of package \(package.name):")
+            errors.forEach { print($0) }
```

Comment:
I think there are couple of errors/warnings which outputs like this which I hope can be refactored later when we have a better diagnostics infrastructure.